src/doc/trpl/enums.md

```diff
@@ -2,67 +2,76 @@
 
 Rust has a ‘sum type’, an `enum`. Enums are an incredibly useful feature of
 Rust, and are used throughout the standard library. An `enum` is a type which
-relates a set of alternates to a specific name. For example, below we define
-`Character` to be either a `Digit` or something else.
+relates a set of alternates to a specific name. Each variant of an
+`enum` is defined like a struct, and can hold any data (or no data, like
+a unit-like struct).
```

Comment:
I'm mostly going for the fact that, syntactically, each variant of an enum is written like one of the ways to write a struct definition. If you're getting at, e.g., people wrongly thinking that you can use a variant of an enum as a standalone thing (like write an impl for a single enum variant), then yeah, I get that distinction. Would rephrasing this to make it clear that it's about syntax, not (so much) semantics, help?